modules/channel-web/src/views/lite/core/api.tsx

```diff
@@ -30,7 +30,7 @@ export default class WebchatApi {
   updateAxiosConfig({ botId = undefined, externalAuthToken = undefined } = {}) {
     this.botId = botId
     this.axiosConfig = botId
-      ? { baseURL: `${window.location.origin}/api/v1/bots/${botId}/mod/channel-web` }
+      ? { baseURL: `${window.location.origin}${window.BOT_API_PATH}/mod/channel-web` }
```

Comment:
so that we could use the SanitzeUrl function above everywhere and be sure we're not breaking and prevent misuse